DEVELOPMENT.md

```diff
@@ -0,0 +1,29 @@
+# Common Color Hex Codes
+
+```css
```

Comment:
Love that you've added this to documentation

Reply:
Me too. Makes sense!